llvm/test/DebugInfo/KeyInstructions/Generic/jump-threading-2-bbs.ll

```diff
@@ -0,0 +1,68 @@
+; RUN: opt -S -passes=jump-threading,verify %s | FileCheck %s
+
+;; Modified from llvm/test/Transforms/JumpThreading/thread-two-bbs.ll
+;;
+;; JumpThreading duplicates bb.cond2 to thread through bb.file to bb.f2 or exit.
+;;
+;; Check the duplicated instructions get remapped atom groups.
+
+; CHECK: bb.cond2:
```

Comment:
CHECK-LABEL?

Reply:
I wouldn't. The point of CHECK-LABEL is to subdivide the checked text; here, it's followed by a series of CHECK-NEXT, so the grouping effect of -LABEL isn't needed.